bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerHandle.java

```diff
@@ -1371,6 +1389,16 @@ void handleUnrecoverableErrorDuringAdd(int rc) {
         asyncCloseInternal(NoopCloseCallback.instance, null, rc);
     }
 
+    void monitorPendingAddOps() {
+        int timedOut = 0;
+        for (PendingAddOp op : pendingAddOps) {
+            if (op.maybeTimeout()) {
+                timedOut++;
+            }
+        }
+        LOG.info("Timed out {} add ops", timedOut);
```

Comment:
maybe it is better to log this only if timedOut > 0, otherwise we will spam logs at INFO level

Reply:
done